lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -232,7 +232,7 @@ sub to_idn {
         return Zonemaster::LDNS::to_idn( $str );
     }
     else {
-        warn __( "Warning: Zonemaster::LDNS not compiled with libidn, cannot handle non-ASCII names correctly." );
+        warn __( "Warning: Zonemaster::LDNS not compiled with libidn2, cannot handle non-ASCII names correctly." );
```

Comment:
I think the following would be better and not bound to details not relevant here.
```
Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle names with non-ASCII characters correctly.
```
But I accept as-is.

Compare with https://github.com/zonemaster/zonemaster-cli/pull/254#discussion_r860800455

Reply:
Good point. I'll do the change in another PR.